src/pages/binnentuin/menu/Menu.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import './Menu.scss';
+import logo_white from '../../../assets/img/logo_white.png';
+
+
+import CardList from "../../../pages/binnentuin/menu/CardList";
+
+class MenuPage extends React.Component {
+    cardClicked = (id) => {
+      console.log(`Gekozen gerecht nummer: + ${id}`);
+    }
+
+    render() {
+      return (
+        <section>
+
+          <header className="header">
+            <section className="container">
+              <figure className="header__figure">
+                <img src={logo_white} alt="Logo van Ruben en Jerry's eetcafé de Binnentuin"/>
+              </figure>
+            </section>
+          </header>
```

Comment:
Hiervoor kan je de Header component importeren, zoals ook in: `src/pages/keuze/Keuze.jsx`

Reply:
done